README.md

```diff
@@ -152,37 +97,37 @@ personal_access_token_arn = data.aws_secretsmanager_secret_version.token.arn
-| ecs_desired_count | 0 | Sets the default desired count for task definitions within the ECS service |
-| assign_public_ip | "false" | Choose whether to assign a public IP address to the Elastic Network Interface |
-| role_path | "/" | The path in which to create the assume roles and policies. Refer to [the AWS docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html) for more |
-| permissions_boundary | "" | ARN of the policy that is used to set the permissions boundary for the role |
-| github_repo_owner | "CMSgov" | The name of the Github repo owner. |
-| tags | {} | Additional tags to apply |
+| Name                     | Default Value | Description                                                                                                                                                              |
+| ------------------------ | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| cloudwatch_log_retention | 731           | Number of days to retain Cloudwatch logs                                                                                                                                 |
+| ecr_repo_tag             | "latest"      | The tag to identify and pull the image in ECR repo                                                                                                                       |
+| ecs_desired_count        | 0             | Sets the default desired count for task definitions within the ECS service                                                                                               |
+| assign_public_ip         | "false"       | Choose whether to assign a public IP address to the Elastic Network Interface                                                                                            |
+| role_path                | "/"           | The path in which to create the assume roles and policies. Refer to [the AWS docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html) for more |
+| permissions_boundary     | ""            | ARN of the policy that is used to set the permissions boundary for the role                                                                                              |
+| github_repo_owner        | "CMSgov"      | The name of the Github repo owner.                                                                                                                                       |
+| tags                     | {}            | Additional tags to apply                                                                                                                                                 |
 
 ### Outputs
 
 None.
 
 ### Requirements
 
-| Name | Version |
-|------|---------|
+| Name      | Version |
+| --------- | ------- |
 | terraform | >= 0.13 |
```

Comment:
Does it make sense to bring the terraform up to a recent version since this is a pre-official release?

Reply:
Yep!  I chose `~> 1.0` for the Terraform version to give us flexibility to update within the production release, let me know what you think.